client/deis.py

```diff
@@ -1271,6 +1266,61 @@ def domains_list(self, args):
         else:
             raise ResponseError(response)
 
+    def git(self, args):
+        """
+        Valid commands for git:
+
+        git:remote          Adds git remote of application to repository
+
+        Use `deis help [command]` to learn more.
+        """
+        return
```

Comment:
Heroku returns the following when you call `heroku git`:

```
Your version of git is 2.1.2. Which has serious security vulnerabilities.
More information here: https://blog.heroku.com/archives/2014/12/23/update_your_git_clients_on_windows_and_os_x
 !    `git` is not a heroku command.
 !    Perhaps you meant `list`.
 !    See `heroku help` for a list of available commands.
```

It might be handy to show similarly helpful information and call os.exit(1) to follow Heroku's workflow.